diff --git a/node_modules/starknet/dist/index.js b/node_modules/starknet/dist/index.js
index 1569354..fb8d569 100644
--- a/node_modules/starknet/dist/index.js
+++ b/node_modules/starknet/dist/index.js
@@ -5832,7 +5832,7 @@ var Account = class extends RpcProvider2 {
     return this.estimateInvokeFee(calls, details);
   }
   async estimateFeeBulk(invocations, details = {}) {
-    const { nonce, blockIdentifier, version } = details;
+    const { nonce, blockIdentifier, version, skipValidate } = details;
     const accountInvocations = await this.accountInvocationsFactory(invocations, {
       ...v3Details(details),
       versions: [
@@ -5845,11 +5845,12 @@ var Account = class extends RpcProvider2 {
         // sierra
       ],
       nonce,
-      blockIdentifier
+      blockIdentifier,
+      skipValidate
     });
     return super.getEstimateFeeBulk(accountInvocations, {
       blockIdentifier,
-      skipValidate: details.skipValidate
+      skipValidate: skipValidate
     });
   }
   async simulateTransaction(invocations, details = {}) {
